course-definition.yml

```
@@ -127,6 +127,18 @@ extensions:
       [redis-pub-sub]: https://redis.io/docs/latest/develop/pubsub/
       [subscribe-command]: https://redis.io/docs/latest/commands/subscribe/
       [publish-command]: https://redis.io/docs/latest/commands/publish/
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)